### PR TITLE
Column creation

### PR DESCRIPTION
**Pull request by nisHcodeX**

### CODE REVIEW

File: src/components/icons/index.tsx [hunk 1]
```diff
@@ -1,3 +1,8 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+
 export const IconPlus = ({ size = "md" }: any) => (
     <span className={`k-icon k-icon-${size} k-i-plus-outline `}></span>
 );
+
+export const IconFa = ({ icon }) => <FontAwesomeIcon icon={icon} />
```
Comment (rishangit) [inline src/components/icons/index.tsx:8]: @nisHcodeX  Need to have fixed exported component for edit and deleted 

export const IconEdit  = () => <FontAwesomeIcon icon={ faPen } />
export const IconDelete= () => <FontAwesomeIcon icon={ faTrashCan } />

File: src/modules/columns/common/column.icons.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import { columnTypeProps } from '@modules/columns/manage/constant';
+import { IconFa } from '@app/icons';
+
+
+export const IconText = ({ keyIcon }) => {
+  const { icon } = columnTypeProps.find(({ type }) => type === keyIcon)
+  return (
+  <ColumnIcon setBorder={true}>
+        <IconFa icon={icon}/>
+    </ColumnIcon>
```
Comment (rishangit) [inline src/modules/columns/common/column.icons.tsx:10]: Need proper alignment for the code

File: src/modules/columns/list/index.tsx [hunk 3]
```diff
@@ -47,31 +53,29 @@ const ListComponent: FC = () => {
...
+                    </h6>
                 </ColumnTableThreads>
                 <ColumnTableThreads span='7'>
                   {raw.type}
                 </ColumnTableThreads>
                 <ColumnTableThreads span='1'>
-                <ColumnIcon>
-                    Ab
+                  <ColumnIcon>
+                    <IconFa icon={faPen} />
```
Comment (rishangit) [inline src/modules/columns/list/index.tsx:65]: IconEdit

File: src/modules/columns/list/index.tsx [hunk 4]
```diff
@@ -47,31 +53,29 @@ const ListComponent: FC = () => {
...
                   </ColumnIcon>
                   <ColumnIcon>
-                    Ab
+                    <IconFa icon={faTrashCan} />
```
Comment (rishangit) [inline src/modules/columns/list/index.tsx:68]: IconDelete

File: src/modules/columns/list/index.tsx [hunk 5]
```diff
@@ -1,11 +1,13 @@
 import { FC, memo, useState } from "react";
 import { useSelector } from "react-redux";
-
+import { faPen, faTrashCan } from '@fortawesome/free-solid-svg-icons'
```
Comment (rishangit) [inline src/modules/columns/list/index.tsx:3]: can remove when use IconEdit and IconDelete

File: src/modules/columns/manage/selectType.tsx [hunk 6]
```diff
@@ -21,12 +20,9 @@ const SelectType = ({ setModel }) => {
         {Object.keys(ColumnType).map((key) => {
```
Comment (rishangit) [inline src/modules/columns/manage/selectType.tsx:23]: foreach on 'columnTypeProps'

File: src/modules/columns/manage/selectType.tsx [hunk 7]
```diff
@@ -1,15 +1,14 @@
 import { ColumnType, Column } from "../types";
-import { ColumnCategory, ColumnIcon, ColumnItem, ColumnData, ColumnTitle } from "@modules/columns/columns.styled";
+import { ColumnCategory, ColumnItem, ColumnData, ColumnTitle } from "@modules/columns/columns.styled";
 import { columnTypeProps } from "@modules/columns/manage/constant";
-// import { Fa } from "react-icons/fa"
-
 import 'bootstrap/dist/css/bootstrap.min.css'
```
Comment (rishangit) [inline src/modules/columns/manage/selectType.tsx]: add bootstrap in to higher file, that can use inside the project

App.tx

File: src/modules/columns/manage/selectType.tsx [hunk 8]
```diff
@@ -21,12 +20,9 @@ const SelectType = ({ setModel }) => {
         {Object.keys(ColumnType).map((key) => {
           return <ColumnItem key={key} onClick={() => onSelect(key)}>
             <ColumnData flexDirection='row'>
-              <ColumnIcon >
-                <h4 style={{ margin: '0' }}>
-                  Ab
-              </h4>
-              </ColumnIcon>
-            </ColumnData>
+              <IconText keyIcon={key} />
```
Comment (rishangit) [inline src/modules/columns/manage/selectType.tsx]: <ColumnIcon type={ type } />

File: src/components/common/window/index.tsx [hunk 9]
```diff
@@ -21,7 +22,7 @@ const PopupWindow: FC<PopupWindowProps> = ({
 
   return (
     <>
-      {showOverLay && <div className="k-overlay"></div>}
+      {showOverLay && <PopupShade></PopupShade>}
```
Comment (rishangit) [inline src/components/common/window/index.tsx]: @nisHcodeX  self closing component 

<PopupShade/>

File: src/modules/columns/manage/selectType.tsx (hunk 6 above)
Comment (nisHcodeX) [reply src/modules/columns/manage/selectType.tsx:23]: I use the map function

**Pull request merged**